### PR TITLE
feat: mark static text in highlights page for i18n

### PR DESCRIPTION
**Pull request by muhammad-ammar**
**Description:** Mark static text on highlights page for i18n.

**JIRA:** https://2u-internal.atlassian.net/browse/ENT-8753

# For all changes

- [ ] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/ContentHighlights/DeleteHighlightSet.jsx [hunk 1]
```diff
@@ -128,15 +154,31 @@ const DeleteHighlightSet = ({ enterpriseId, enterpriseSlug }) => {
...
+            />
           </p>
         </Alert>
         <p>
-          Deleting this highlight will remove it from your
-          learners&apos; &quot;Find a Course&quot;. This action is permanent and cannot be undone.
+          <FormattedMessage
+            id="highlights.modal.delete.highlight.confirmation.message"
+            defaultMessage="Deleting this highlight will remove it from your learners{apostrophe} {doubleQoute}Find a Course{doubleQoute}. This action is permanent and cannot be undone."
+            description="Confirmation message shown when deleting a highlight."
```
Comment (jajjibhai008) [inline src/components/ContentHighlights/DeleteHighlightSet.jsx:176]: can't we directly mention apostrophe and doubleQoute? what kind of problem we are getting?

**Review by jajjibhai008 — APPROVED** [review]

File: src/components/ContentHighlights/DeleteHighlightSet.jsx (hunk 1 above)
Comment (muhammad-ammar) [reply src/components/ContentHighlights/DeleteHighlightSet.jsx:176]: this text contains both the single and double quotes. Either we should escaping by `\` or use `values`

File: src/components/ContentHighlights/CatalogVisibility/ContentHighlightCatalogVisibilityAlert.jsx [hunk 2]
```diff
@@ -74,15 +83,27 @@ const ContentHighlightCatalogVisibilityAlert = () => {
               data-testid={`catalog-visibility-alert-${BUTTON_TEXT.catalogVisibility}`}
```
[the remark this answers is not in the document]
Comment (muhammad-ammar) [reply src/components/ContentHighlights/CatalogVisibility/ContentHighlightCatalogVisibilityAlert.jsx:83]: We can change this but the value of `BUTTON_TEXT.catalogVisibility` is `New highlight`
So the complete test id will become `catalog-visibility-alert-New highlight` which looks weird to me du to spaces?

[the remark this answers is not in the document]
Comment (jajjibhai008) [reply src/components/ContentHighlights/CatalogVisibility/ContentHighlightCatalogVisibilityAlert.jsx:83]: [optional] we can place a dash here like `catalog-visibility-alert-new-highlight` and update the test accordingly.

**Pull request merged**